updatecli/updatecli.d/charts/jenkins-kubernetes-agent.yaml

```diff
@@ -20,21 +20,30 @@ sources:
       name: jenkins-kubernetes-agents
 
 targets:
-  updateChartVersion:
+  updateCik8sChartVersion:
     name: "Update the chart version for jenkins kubernetes agent"
     kind: file
     spec:
       file: clusters/cik8s.yaml
       matchPattern: 'chart: jenkins-infra\/jenkins-kubernetes-agents((\r\n|\r|\n)(\s+))version: .*'
       replacePattern: 'chart: jenkins-infra/jenkins-kubernetes-agents${1}version: {{ source `lastChartVersion` }}'
     scmID: default
+  updateDoksChartVersion:
+    name: "Update the chart version for jenkins kubernetes agent"
+    kind: file
+    spec:
+      file: clusters/doks.yaml
+      matchPattern: 'chart: jenkins-infra\/jenkins-kubernetes-agents((\r\n|\r|\n)(\s+))version: .*'
+      replacePattern: 'chart: jenkins-infra/jenkins-kubernetes-agents${1}version: {{ source `lastChartVersion` }}'
+    scmID: default
```

Comment:
Damn, the current "updatecli diff" process is blocking the whole "helmfil diff" on the build for the PR.
Do you mind if we comment out the code changed in this updatecli manifest for this PR and uncomment on na subsqeunt PR?

Reply:
WDYT about spliting out updatecli to its own pipeline?

Reply:
That way out of the scope of this PR: yes if you are willing to it somewhere else but let's start by finishing this PR first :)